content/en/opentelemetry/otel_collector_datadog_exporter.md

```diff
@@ -99,22 +102,131 @@ The above configuration enables the receiving of OTLP data from OpenTelemetry in
 
```

Comment:
The example above (starting at line 36) shows metrics and traces only. I think we should include the basic logs config too (even though it's also further discussed below). The basic logs config has a [filelog section](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/examples/collector.yaml#L52) and a [service.pipeline section](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/examples/collector.yaml#L303).

Reply:
@kayayarai Added the config sections, but I left out the commented sections, should we include those?

Reply:
@estherk15 no, I would leave out the commented sections, same as what you did